src/App.css

```diff
@@ -36,3 +37,108 @@
     transform: rotate(360deg);
   }
 }
+
+/* home banner */
+.banner-imgs{
+  width: 90%;
+  height: 500px;
+  opacity: 0.6;
+  position: relative;
+}
+.banner-text{
+  position: absolute;
+  top: 13rem;
+  left: 5rem;
+  right: 5rem;
+  font-size: 3rem;
+}
+
+/* nav bar */
+.nav-bar{
+  background-color:#00416A;
+}
+
+/* Video card list */
+
+.video-card-cont{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.video-card{
+  display: block;
+  border-radius: 1rem;
+  padding: 1rem;
+  width: 50rem;
+  margin: 1rem;
+  background-color: whitesmoke;
```

Comment:
use variable for colors

Reply:
Actually this color has been used only once, so not used variable